CHANGELOG.next.asciidoc

```diff
@@ -199,6 +199,7 @@ https://github.com/elastic/beats/compare/v8.8.1\...main[Check the HEAD diff]
 - Introduce log message for not supported annotations for Hints based autodiscover {pull}38213[38213]
 - Add persistent volume claim name to volume if available {pull}38839[38839]
 
```

Comment:
extra line here?

Reply:
With all the rebases and merges this PR has been through, this line appeared there. I'll remove it with the next commit.

Reply:
Done 4673969702